core/src/main/scala/ai/lum/odinson/ExtractorEngine.scala

```diff
@@ -81,11 +83,11 @@ class ExtractorEngine(
 
   /** executes query and returns next n results after the provided doc */
   def query(
-    odinsonQuery: String,
-    n: Int,
-    afterDoc: Int,
-    afterScore: Float
-  ): OdinResults = {
+             odinsonQuery: String,
```

Comment:
change idea's tab formatting to 2 spaces and save to fix this indendation

Reply:
It wasn't tabs just different formatting style (in this case for multi line function definition). Added scalafmt to the project (code style format) and formatted the files with it, the function definitions are now as they were before but it also changed other things but as long as its standard and not incidental, I'm ok with it.